content/blog/bitigchi/2020-10-01-haiku_monthly_activity_report_september_2020.md

```diff
@@ -0,0 +1,96 @@
++++
+type = "blog"
+title = "Haiku Activity Report - September 2020"
+author = "bitigchi"
+date = "2020-10/01 13:00:00+03:00"
+tags = ["monthly activity report"]
++++
+
+Welcome to the Haiku Monthly Activity Report for September 2020. This report covers revisions from hrev54539 to hrev54608.
+
+## System Sounds Contest
+
+We've started a contest for the system sounds, and currently it's ongoing. For more details, see the contest post.
+
+## Applications
+
+AlwaysLivid improved the after-install UX via a bunch of Installer and FirstBootPrompt fixes. Now it is possible to exit the FirstBootPrompt to Desktop without having to restart the system.
+
+AlwaysLivid added Mail auto-configure data for mailbox.org and riseup.net.
+
+bitigchi fixed the redundant reboot confirmation in SoftwareUpdater.
+
+Pascal Abresch made TeamMonitor and AboutSystem compliant to the system colours; previously the text colours would not react properly to dark system themes.
+
+PulkoMandy fixed various issues with MediaPlayer and ffmpeg implementation that caused seeking and cover art display problems.
+
+Murai Takashi made some code cleanups in various system parts.
+
+syedsouban and bitigchi implemented custom search engine shortcuts in WebPositive. Currently following shortcuts are supported:
+
+- Google (g)
+- Bing (b)
+- Wikipedia (w)
+- DuckDuckGo (d)
+- Baidu (a)
+- Yandex (y)
+- Ecosia (e)
+- Qwant (q)
+
+## Compatibility
+
+CodeForEvolution implemented the `qsort_r` function.
+
+## Documentation
+
+John Scipione improved the documentation for BMessage and AppKit.
+
+nielx implemented layout customisations for the Doxygen stylesheet.
+
+## File Systems
+
+brjhaiku implemented the initial methods required for btrfs write support.
+
+## Kernel
+
+Jerome Duval implemented support for more CPU power modes and functions that allows system to use the CPU power more efficiently. This fixed an issue where on some hardware the system clock speed was stuck at 1.5 GHz, and improved performance on other systems.
+
+## Package Management
+
+Andrew Lindesay fixed a crash in HaikuDepot whilst synchronising package metadata.
```

Comment:
I think https://git.haiku-os.org/haiku/commit/?id=hrev54558 should also be mentioned here.